react-hot-button-component/my-app/src/HotButton.jsx

```diff
@@ -0,0 +1,45 @@
+import './styles.css';
+import { useState } from 'react'
+
+export default function HotButton() {
+  const [count, setCount] = useState(0);
+
+  let colorBg = 'indigoBg';
+  let colorText = 'whiteText';
+
+  switch (Math.floor(count/3) % 6) {
+    case 0:
+      colorBg = 'indigoBg';
+      colorText = 'whiteText';
+      break;
+    case 1:
+      colorBg = 'purpleBg';
+      colorText = 'whiteText';
+      break;
+    case 2:
+      colorBg = 'pinkBg';
+      colorText = 'whiteText';
+      break;
+    case 3:
+      colorBg = 'orangeBg';
+      colorText = 'blackText';
+      break;
+    case 4:
+      colorBg = 'yellowBg';
+      colorText = 'blackText';
+      break;
+    case 5:
+      colorBg = 'whiteBg';
+      colorText = 'blackText';
+      break;
+    default:
+      break;
+  }
+
+  return (
+  <div>
+    <button className = {`${colorBg} ${colorText}`} onClick = {() => setCount(count + 1)}>Hot Button</button>
```

Comment:
Why not just apply both the background color and text color styling to the same class? That way you don't need the two different variables, and you will just have 1 class